skatepark/sass/blocks/_buttons.scss

```diff
@@ -21,20 +21,23 @@
 }
 .wp-block-file .wp-block-file__button {
 	text-transform: uppercase;
+	letter-spacing: 0.1em;
 	@include skatepark_button_hover_styles;
 }	
 
 .wp-block-search {
 	&.wp-block-search__button-inside .wp-block-search__inside-wrapper .wp-block-search__button,
-	.wp-block-search__button {
+	.wp-block-search__button.wp-block-search__button {
```

Comment:
Why do we need the double selector?

Reply:
[Because of GB](https://github.com/WordPress/gutenberg/blob/d011a7874d82cfa740cb208b42582d824445ced0/packages/block-library/src/search/editor.scss)! I think the block shouldn't be using the components classes, I'll make a PR for it